test/LibraryManager.IntegrationTest/VisualStudioLibmanHostTests.cs

```diff
@@ -17,9 +18,11 @@ namespace Microsoft.Web.LibraryManager.IntegrationTest
     public class VisualStudioLibmanHostTest : VisualStudioHostTest
     {
         // Solution consts
-        const string _testSolutionName = @"TestSolution.sln";
+        const string _libman = "libman.json";
```

Comment:
put access modifiers on all of these.  Probably `protected`?

Reply:
That's on the new ones... I think the existing privates should probably remain so.

Reply:
Updated in iteration 2

---
In reply to: [208033555](https://github.com/aspnet/LibraryManager/pull/341#discussion_r208033555) [](ancestors = 208033555)

Reply:
Updated in iteration 2

---
In reply to: [208033443](https://github.com/aspnet/LibraryManager/pull/341#discussion_r208033443) [](ancestors = 208033443)